pkg/controlplane/apiserver/options/options.go

```diff
@@ -262,6 +269,23 @@ func (o *Options) Complete(alternateDNS []string, alternateIPs []net.IP) (Comple
 			if err != nil {
 				return CompletedOptions{}, fmt.Errorf("failed to build token generator: %w", err)
 			}
+		} else if completed.ServiceAccountSigningEndpoint != "" {
```

Comment:
can make sure this is mutually exclusive here as well:

```go
switch {
case completed.ServiceAccountSigningKeyFile != "" && completed.ServiceAccountSigningEndpoint != "":
  ... error
case completed.ServiceAccountSigningKeyFile != "":
 ...
case completed.ServiceAccountSigningEndpoint != "":
 ...
}
```

Reply:
Done